tests/lib/cli.js

```diff
@@ -18,7 +18,6 @@ const assert = require("chai").assert,
     CLIEngine = require("../../lib/cli-engine/index").CLIEngine,
     path = require("path"),
     sinon = require("sinon"),
-    leche = require("leche"),
```

Comment:
is `leche` still used somewhere else?(if no, we can remove it from `package.json`)

Reply:
Yes, 5 test files still use `leche`.